controllers/snapshot/snapshot_adapter.go

```diff
@@ -76,10 +76,11 @@ func scenariosNamesToList(integrationTestScenarions *[]v1beta1.IntegrationTestSc
 	return &result
 }
 
-// EnsureAllIntegrationTestPipelinesExist is an operation that will ensure that all Integration test pipelines
+// EnsureIntegrationPipelineRunsInRootEnvsExist is an operation that will ensure that all Integration pipeline runs
```

Comment:
While this is name is definitely much better than the old one, I'm thinking that `InRootEnvs` might give the false impression that those pipelineRuns have access to modify/deploy/execute things in those environments.
Would `EnsureStaticIntegrationPipelineRunsExist` maybe work better for this?

Reply:
Thanks!